src/table/components/TablePaginationActions.jsx

```diff
@@ -51,31 +55,42 @@ export default function TablePaginationActions({
         aria-label={translator.get(`SNTable.Pagination.${type}`)}
         title={!constraints.passive && translator.get(`SNTable.Pagination.${type}`)}
         tabIndex={tabIndex}
-        sx={disabledCondition ? { color: 'rgba(0, 0, 0, 0.3)', cursor: 'default' } : { color: 'rgba(0, 0, 0, 0.54)' }}
+        sx={disabledCondition ? { color: disabledIconButtonColor, cursor: 'default' } : { color: iconButtonColor }}
         onKeyDown={onKeyDown}
       >
         <IconComponent />
       </IconButton>
     );
   };
 
+  const inputProps = {
+    'data-testid': 'pagination-dropdown',
+    tabIndex,
+    id: 'pagination-dropdown',
+    style: {
+      color: theme.isBackgroundDarkColor ? 'rgba(255, 255, 255, 0.9)' : '#404040',
+    },
+  };
+
+  const selectStyle = {
+    backgroundColor: 'inherit',
+    '& .MuiNativeSelect-icon': {
+      color: theme.isBackgroundDarkColor ? 'rgba(255, 255, 255, 0.9)' : 'rgba(0, 0, 0, 0.54)',
```

Comment:
This one should be iconButtonColor right?

Reply:
Right, good find. Thanks.

Reply:
#510